app/routes/project/index.js

```diff
@@ -10,15 +10,13 @@ export default Route.extend({
   userSubscriptions: service(),
 
   model() {
-    let project = this.modelFor('project');
-
-    return RSVP.hash({
-      project,
-      subscription: this.get('userSubscriptions').fetchForProject(project)
+    return this.modelFor('project').reload().then((project) => {
```

Comment:
🤔 This seems a bit like an anti-pattern to me. Typically, I've seen the `reload` method used outside of the context of the model hook, to force the model to reload because of an action (like in the example in the docs: http://emberjs.com/api/data/classes/DS.Model.html#method_reload).

Is there a reason we don't just write the model hooks we need for these child routes rather than forcing our children to trigger an update on our parent?

Rather than relying on what is returned from our parent route, can we just call the `project` model again here?  That will update the record in our store, which will ultimately update our parent's route model as well.

Thoughts?